lib/composables/Image.ts

```diff
@@ -13,7 +13,13 @@ export interface ImageSrcFromFile {
 export function useImageSrcFromFile(
   file: MaybeRefOrGetter<File | string | null>
 ): ImageSrcFromFile {
-  const reader = new FileReader()
+  if (typeof document === 'undefined') {
```

Comment:
Should we add support function for this? 🤔  This could be common scenario when supporting SSR.

Something like:

```ts
// support/Utils.ts
// or `inBrowser`...? I think I saw other frameworks doing the
// similar thing so would like to follow though.
export function isBrowser(): boolean
```

Reply:
We can just use isClient from vueuse 😅

Reply:
Oh OK hooray to VueUse 👍 They have everything 😆